### PR TITLE
High performance Draw3DLine function

### PR DESCRIPTION
**Pull request by manhnt9**
It speeds up my ray debug rendering so much so I can't help firing a pull request.

### CODE REVIEW

File: ext/DebugDraw/CDraw3DLine.cpp [hunk 1]
```diff
@@ -71,6 +72,32 @@ void CDraw3DLine::draw(
     fence->drop();
 }
 
+void CDraw3DLine::draw(const core::vector<S3DLineVertex>& linesData)
```
Comment (devshgraphicsprogramming) [inline ext/DebugDraw/CDraw3DLine.cpp]: should that be a vector of std::pair ?

Comment (devshgraphicsprogramming) [reply ext/DebugDraw/CDraw3DLine.cpp]: since you will always require an even number of vertices, make that a vector of vertex-pairs

Comment (manhnt9) [reply ext/DebugDraw/CDraw3DLine.cpp]: I will update soon

**Comment by manhnt9** [comment]
Well something made this PR automatically closed ...

**Comment by manhnt9** [comment]
Ah I lost my changes, maybe I reset my branch without caring about this PR. Anyways, the code is still in my engine so will make a new PR before new week starts.

**Pull request closed without merging**